### ISSUE TITLE
Switch format of local nodes environment variable

### ISSUE THREAD

**Issue by alyssadai** [neurobagel/federation-api#32]
`( )` characters have special meaning in bash (arrays), so using them requires the use of a quoted string which may create problems for `docker run`.

**Comment by surchs** [comment]
Agreed, and I think we should decide on this before we release the new f-API docs on users in https://github.com/neurobagel/documentation/issues/129.

I like the JSON idea since that's what we're already using for https://github.com/neurobagel/menu/blob/main/node_directory/neurobagel_public_nodes.json. So then the requirements for deployment would be:

- the `local_nodes.json` file
- a `docker-compose.yml` file to launch both the f-API and the query tool

and that's it?

**Comment by surchs** [comment]
@alyssadai see if the updates spec is more what you had in mind and feel free to add more options. Once the spec is clear, I'd say we discuss which option to pick and then move it to Specify-Done

**Comment by alyssadai** [comment]
I've added an additional point to consider under each of the alternatives. Other than that, I think those are the options I had in mind. I'm still leaning towards the comma-separated pairs option (with no spaces in the node name), but let me know what you think @neurobagel/dev.

**Comment by rmanaem** [comment]
Decision: Go with a `.json` file for storing configuration.

**Comment by surchs** [comment]
:rocket: Issue was released in [`v0.1.0`](https://github.com/neurobagel/federation-api/releases/tag/v0.1.0) :rocket:
